src/main/java/com/netgrif/application/engine/petrinet/domain/version/Version.java

```diff
@@ -11,7 +11,10 @@
 @AllArgsConstructor
 public class Version {
 
-    public static final String NEWEST = "latest";
+    @Deprecated
```

Comment:
The `@Deprecated` annotation should specify what to replace the constant with (it should link to the new `LATEST` constant).

Reply:
Done